bedrock/firefox/templates/firefox/mobile-download-desktop.html

```diff
@@ -49,21 +54,32 @@
             <h2>{{ high_res_img('firefox/template/logo-large.png', {'alt': 'Firefox', 'width': '136', 'height': '142'}) }}</h2>
           </header>
           <div class="main-content">
+          {% block headline %}
             <h1>
-            {% if LANG.startswith('en-') %}
-              {# Custom headline for en-* users #}
-              Send Firefox to your phone and<br> set your mobile browsing free
+            {# If user is in a locale with translated basket messages... #}
+            {% if show_send_to_device %}
+              {# Custom headlines for en-* users #}
+              {% if LANG.startswith('en-') %}
+                Send Firefox to your phone and set your mobile browsing free
+              {# Headline for non en-* users in locales with translated basket messages #}
+              {% else %}
+                {{ _('Send Firefox to your smartphone or tablet') }}
+              {% endif %}
             {% else %}
-              {# Pre-localized headline for all other users (taken from send to device widget) #}
-              {{ _('Send Firefox to your smartphone or tablet') }}
+              {# Generic, pre-localized headline #}
+              {{ _('Download Firefox') }}
             {% endif %}
             </h1>
+          {% endblock %}
             <div class="content">
+            {# Hide the widget if user is not in a supported basket locale #}
+            {% if show_send_to_device %}
               {{ send_to_device(include_title=False, message_set='fx-mobile-download-desktop') }}
+            {% endif %}
 
               <ul id="mobile-download-buttons">
                 <li class="android">
-                  {{ google_play_button(alt_href='https://app.adjust.com/2uo1qc?campaign=Heartbeat&adgroup=Google_Play_Store&creative=badge') }}
+                  {{ google_play_button(alt_href='https://app.adjust.com/2uo1qc?campaign=Heartbeat&adgroup=Google_Play_Store&creative=badge', anchor_attributes={'data-link-type': 'download', 'data-download-os': 'Android'}) }}
```

Comment:
I *think* the idea is that if it's not the standard Google Play link, we should also re-consider the tracking attributes (which happened to be the same in this instance). I would say if we have one more situation like this (alternate URL, same tracking attrs), we re-work the helper.